maquetacion/book-detail.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="./dist/css/application.css" />
+  </head>
+
+  <body>
+
+    <nav id="nav" class="nav">
+      <div class="container">
+        <div class="nav__content">
+          <img src="./assets/logo-wolox.png" alt="Logo Wolox" class="nav__logo">
+          <a href="/index.html" class="nav__logout">Logout</a>
+        </div>
+      </div>
+    </nav>
+    
+    <div class="container content">
+
+      <div class="back-button">
```

Comment:
I think this `div` is not needed. If you need the styles, applied it to the `a` tag 😄